client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/entity/CarColor.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.common.domain.entity
+
+enum class CarColor(val hexadecimal: Long) {
+    RED(0xffF47373),
+    YELLOW(0xffF8EC7E),
+    GREEN(0xff80E5AB),
+    BLUE(0xff77DEEE),
+    WHITE(0xffFFFFFF),
+    GREY(0xffAFAFAF),
+    BLACK(0xff3F3F3F)
+}
```

Comment:
there is no need to specify the value of colors
for example if we wanted to change the colors shade in dark mode this won't help us
also all the colors that we use are in design system in theme and this is a ui layer so if you can add these colors in design system and just map the enum and let the ui decide what its value